auth0/resource_auth0_log_stream.go

```diff
@@ -164,6 +176,8 @@ func readLogStream(d *schema.ResourceData, m interface{}) error {
 	d.Set("status", c.Status)
 	d.Set("type", c.Type)
 	d.Set("sink", flattenLogStreamSink(d, c.Sink))
+	d.Set("aws_partner_event_source", logStreamAwsPartnerEventSource(c.Sink))
```

Comment:
@abulford  this code is already implemented in flattenLogStreamEventGridSink and flattenLogStreamEventBridgeSink respectively.